dev-util/pycharm-professional/pycharm-professional-2018.3.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils readme.gentoo-r1 xdg
```

Comment:
Just for transparency:
https://github.com/gentoo/gentoo/pull/9373#discussion_r205968485